ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.html

```diff
@@ -541,6 +541,18 @@ <h3 class="Color--primary Line--Break">{{additionalMessage}}</h3>
                     </ng-container>
                   </div>
 
+                  <div *ngIf="col.participantColumn.tableAlias==='data' && col.participantColumn.object==='files'">
+                    <ul class="NO-PADDING-START">
+                      <li class="LIST-SEPARATOR" *ngFor="let file of pt.data.files">
+                        <p class="BIT-OF-MARGIN">
+                          <ng-container>
```

Comment:
I can not see any benefits for using ng-container here.

Reply:
As it seems, I unknowingly tend to over use them. Thanks for catching that!

Reply:
Often it is used when you have more than 1 structural directive on an element, e.g. *ngIf and *ngFor
see [examples](https://www.digitalocean.com/community/tutorials/angular-ng-container-element )